src/Indice.AspNetCore.Identity/Features/PushNotifications/DevicesController.cs

```diff
@@ -67,12 +69,12 @@ internal class DevicesController : ControllerBase
         [HttpGet]
         [ProducesResponseType(statusCode: StatusCodes.Status200OK, type: typeof(IEnumerable<DeviceInfo>))]
         [ProducesResponseType(statusCode: StatusCodes.Status404NotFound, type: typeof(ProblemDetails))]
-        public async Task<IActionResult> GetDevices() {
+        public async Task<IActionResult> GetDevices([FromQuery] ListOptions<UserDeviceFilter> options = null) {
```

Comment:
Since you added ListOptions parameter on the endpoint, please return a ResultSet and use the ToResultSetAsync evaluation method instead of ToArrayAsync

Reply:
Response sample from Postman after changing to `ToResultSetAsync`

```js
{
    "count": 1,
    "items": [
        {
            "deviceId": "4ACEA1B7-3E47-4F08-9AB6-110499D1DE91",
            "deviceName": "Android Sql",
            "devicePlatform": "WindowsPhone",
            "isPushNotificationsEnabled": true
        }
    ]
}
```